lesson5/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href="/lesson5/styles.css" rel="stylesheet">
+  <link rel = "preconnect" href = "https://fonts.googleapis.com">
+  <link rel = "preconnect" href = "https://fonts.gstatic.com" crossorigin>
+  <link href = "https: //fonts.googleapis.com/css2? family = Roboto & display = swap "rel =" stylesheet ">
+</head>
+<body>
+ <div class="footer">
+  <div class="row-container">
+    <ul class="About-items">
+     <li class="text">About</li>
+     <li class="text">How Triphouse works</li>
+     <li class="text">Careers</li>
+     <li class="text">Privacy</li>
+     <li class="text">Terms</li>
+    </ul>
+    <ul class="Property-items">
+      <li class="text">Property types</li>
+      <li class="text">Guest houses</li>
+      <li class="text">Hotels</li>
+      <li class="text">Apartments</li>
+      <li class="text">Villas</li>
+      <li class="text">Holiday homes</li>
+      <li class="text">Hostels</li>
+    </ul>
+    <ul class="Support-items">
+      <li class="text">Support</li>
+      <li class="text">Contact Customer Service</li>
+      <li class="text">FAQ</li>
+    </ul>
+  </div>
+  <div class="All-items">
+      <li class="text All">&#169 2020 Triphouse, lnc. All rights reserved</li>
+  </div>
```

Comment:
It should be inside the footer tag because it is part of it.
all-items is not an obvious name for the class. You can use "copyright"
also li should only be used inside ul. Can use div or p here.

Reply:
Исправила)